ParkingLot/ParkingAttendant/ParkingStrategy/IParkingStrategy.cs

```diff
@@ -0,0 +1,10 @@
+﻿using ParkingLot.ParkingBase;
+using System.Collections.Generic;
+
+namespace ParkingLot.ParkingAttendant.ParkingStrategy
+{
+    public interface IParkingStrategy
+    {
+        string ParkingCar(string car, List<Parking> parkinglotList);
```

Comment:
Should the current strategy just select the parking lot and not involve parking actions?